README.md

```diff
@@ -16,7 +16,17 @@ cp setup/validators.txt ~/.ziggurat/ripple/setup
     path = "<path to the directory where you built rippled>"
     start_command = "./rippled"
     ```
-5. Run tests with `cargo +stable t -- --test-threads=1`.
+5. Create a package of IP addresses which are required for performance tests. From the root repository directory run, e.g.:
+   Under Linux (to generate dummy devices with addresses):
+   ```
+   sudo python3 ./tools/ips.py --subnet 1.1.1.0/24 --file src/tools/ips.rs --dev_prefix test_zeth
+   ```
+   Under MacOS or Linux (to add whole subnet to loopback device - under Linux: lo, MacOS: lo0):
```

Comment:
oh, right, we can probably remove the below part:
```
##### Mac users
Make sure these two `127.0.0.x` (where `x != 1`) addresses are enabled:

    sudo ifconfig lo0 alias 127.0.0.2 up;
    sudo ifconfig lo0 alias 127.0.0.3 up;
```
On line 39. I cannot leave a comment there...

Reply:
Not yet, VALIDATOR_IPS are hardcoded local IPs (three addresses) used in some other places (like c024). Currently it's demanded to be exactly those two additional addresses and they cannot be from any other subnet (that could  be changed by refactoring the tests). 
So in this PR I'd leave it as is.